src/main/scala/org/broadinstitute/clio/model/authorization.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.clio.model
+
+import akka.http.scaladsl.model.headers.{ModeledCustomHeader, ModeledCustomHeaderCompanion}
+
+import scala.util.Try
+
+
+trait OidcHeader {
```

Comment:
Re the naming _only_: from the [clio wiki](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509#Clio(MetadataManager)-Filenaming) and borrowed from the [scala style guide](http://docs.scala-lang.org/style/files.html):

> As a rule, files should contain a single logical compilation unit. … One exception to this guideline is for classes or traits which have companion objects. … These files should be named according to the class, trait or object they contain …
> All multi-unit files should be given camelCase names with a lower-case first letter. This is a very important convention. It differentiates multi- from single-unit files, greatly easing the process of finding declarations.